browser/src/UI/components/Tabs.tsx

```diff
@@ -183,10 +183,13 @@ const getTabsFromVimTabs = createSelector(
 
 const mapStateToProps = (state: State.IState, ownProps: ITabContainerProps): ITabsProps => {
 
-    const shouldUseVimTabs = state.configuration["tabs.showVimTabs"]
+    const oniTabMode = state.configuration["tabs.mode"]
+    const oniTabsEnabled = state.configuration["tabs.enabled"]
+
+    const shouldUseVimTabs = oniTabMode === "tabs"
     const tabs = shouldUseVimTabs ? getTabsFromVimTabs(state) : getTabsFromBuffers(state)
 
-    const visible = state.configuration["tabs.enabled"]
+    const visible = oniTabsEnabled && oniTabMode !== "native"
```

Comment:
I think that there is one case we'd still need it - if the user wants to disable the tab line completely. I want to make sure there is an easy way to 'turn off' the tab line. It looks like our current set of options support buffers, tabs, and 'native' vim rendering - but not a clear way to turn off the tabline.

Turning off / hiding the tabline is different than setting `tabs.mode` to `'native'`, because in that case, we turn off `ext_tabline`, and hand over the responsibility of rendering the tabline back to Vim - so it doesn't actually 'turn it off', just lets Vim render it. So what we're missing currently is that 'turn off' or 'hide' story.

I can see two ways to preserve this 'hide' functionality:
-  __Keep `tabs.enabled`__- it'd basically be the same as your changes today, except we'd want the `ext_tabline` option to be controlled only by the `native` setting - if `tabs.enabled` is `false`, we'd want to keep it 'externalized' (so vim doesn't try and render it, and it'd be hidden). 
- __Remove `tabs.enabled`, add a `'hidden'` or `false` value to `tabs.mode`__ - when the `hidden` value is set, we'd keep `ext_tabline` set to `true`, but not render the Oni tab UI.

I'm okay with either approach, whichever one is more intuitive - the second one means we can completely  remove the `tabs.enabled` setting, and address confusion about how `tabs.enabled` and `tabs.mode` set to `native` relate, so it seems like that might be better.

Basically I want to make sure if someone asks "How do I turn off the tabline?" we have a clear story for doing so 😄

Reply:
Ahhh okay,  I was forgetting the use case of not wanting a tabline of any fashion, I was only considering Oni Buffer/Oni Tab/Normal Vim Tabline.

I'll push that up in a mo.